### PR TITLE
EZR-9877 Upgrade coding-standard deps

### PR DESCRIPTION
**Pull request by fredsted**

### CODE REVIEW

**Review by admarc — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -41,4 +41,4 @@
     }
   },
   "version": "1.0"
```
Comment (brzuchal) [inline composer.json]: I guess we should bump that too.

**Review by amodliszewski — APPROVED** [review]

**Comment by amodliszewski** [comment]
Did someone checked differences in doctrine standard? We are bumping from 5 to 8, something might be backwards incompatible, and could require changes on our side...

**Comment by admarc** [comment]
Thats actually true - we might not want some of the changes and apply them in zebra from the beginning. Would nice to check what are the differences and maybe discuss them ?

**Review by admarc — CHANGES_REQUESTED** [review]
Please see latest comments.

**Comment by fredsted** [comment]
@admarc  @amodliszewski What do you expect here? A complete list of changes? Do we just never upgrade then?

**Comment by brzuchal** [comment]
@admarc here you go https://www.doctrine-project.org/projects/doctrine-coding-standard/en/8.2/reference/index.html#introduction

**Review by brzuchal — APPROVED** [review]
I think that is ok to bump these dependencies. Whenever there's a need for a single or subset of rules to be tuned and/or ignore it should be done on project level `phpcs.xml.dist` file.
In all, we'll tag it with the next `1.1.0` version number so there's always a place to adjust version constraint on projects `composer.json`

**Pull request merged**